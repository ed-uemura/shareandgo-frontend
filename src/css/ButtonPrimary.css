.btn-primary {
	background-color: var(--SGGreen);
	border: var(--SGGreen);
	font-size: var(--SGTextAssetSize);
	font-weight: 700;
	border-radius: 25px;
	padding: 12px 0px;
}

.btn-primary:hover {
	background-color: var(--SGDarkGreen);
}
