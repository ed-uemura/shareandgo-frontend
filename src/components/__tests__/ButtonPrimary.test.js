import {render, screen, cleanup} from '@testing-library/react'
import renderer from 'react-test-renderer'
import ButtonPrimary from '../ButtonPrimary';
import DriverImage from '../../images/profile-picture-mockup.jpg';
import { MemoryRouter } from 'react-router-dom';


afterEach(() =>{
    cleanup();
});

test('matches snapshot', () =>{ 
    const text = 'Login'
    const className = 'col-6 col-md-3 mx-auto d-block'
    const link = '/main-passenger'
    const clickAction = ''
    const tree = renderer.create(
        <MemoryRouter>
            <ButtonPrimary text={text} className={className} link={link} clickAction={clickAction}/>
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});