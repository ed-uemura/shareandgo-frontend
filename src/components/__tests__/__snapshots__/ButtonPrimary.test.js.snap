// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<a
  className="btn btn-primary col-6 col-md-3 mx-auto d-block"
  href="/main-passenger"
  onClick={[Function]}
>
  Login
</a>
`;
